src/coprocessor/codec/table.rs

```diff
@@ -313,29 +320,35 @@ impl RowColsDict {
         RowColsDict { value, cols }
     }
 
+    /// Returns the total count of the columns.
+    #[inline]
     pub fn len(&self) -> usize {
         self.cols.len()
     }
 
+    /// Returns whether has columns or not.
+    #[inline]
     pub fn is_empty(&self) -> bool {
         self.cols.is_empty()
     }
 
+    /// Gets a column data from its meta if key exists.
```

Comment:
-> Gets the column data from its meta if a key exists.
or
-> Gets the column data from its meta if `key` exists.
??